components/collections/editRequest.vue

```diff
@@ -93,7 +93,7 @@ export default {
     };
   },
   watch: {
-    "requestUpdateData.collectionIndex": function resetFolderIndex() {
+    "requestUpdateData.collectionIndex": ()=> { //resetFolderIndex Function
```

Comment:
An arrow function wouldn't bind this to the Vue instance. Please revert this change.

Reply:
Woah @jamesgeorge007 You're awesome mate